### DISCUSSION TITLE
Acesso a ata da assembleia 2021

### DISCUSSION THREAD

**Discussion by cecivieira** [apyb/comunidade#12] [Geral]
Bom diaaa!!

https://t.co/YZUm6X47Ah
A ata da última assembleia não tá mais pública e também não achei um repositório de atas no github da APyB.

Como a gente pode ter acesso a ela?

> Estava buscando-a para propor o uso do fundo criado com o dinheiro que sobrou da pybr2021.

**Comment by rougeth** [comment]
@cecivieira tenta acessar novamente. Vou discutir com a @apyb/diretoria sobre onde publicar as atas além do Google Drive. Eu queria evitar ter coisas no Drive da APyB com permissões diferentes (leitura a partir do link vs acesso logado). A longo prazo, fica difícil manter.

**Reply by cecivieira** [reply]
Consegui acessar.
Obrigada!!!
